packages/Toast/doc.mdx

```diff
@@ -0,0 +1,259 @@
+---
+name: Toast
+route: /components/toast
+menu: Components
+---
+
+import { Props, Playground } from 'docz'
+import { Code, Dependencies, TagVersion, Usage } from '../../docz'
+import { useToast } from './'
+const { dependencies, peerDependencies, component, name, version } = require('./package.json')
+
+# Toast
+
+<TagVersion version={version} name={name} />
+
+Notification component with [toasted-note](https://toasted-notes.netlify.com) and 2 components : Growl and Snackbar.
+
+## Install and import
+
+<Usage component={component} name={name} peerDependencies={peerDependencies} />
+
+## useToast
+
+Use `useToast` hook to show the toast element. Example below with a custom component.
+
+<Playground>
+  {() => {
+    const toast = useToast()
+    return (
+      <Button
+        onClick={() =>
+          toast(
+            <Box
+              backgroundColor="light.900"
+              borderWidth="1px"
+              borderStyle="solid"
+              borderColor="nude.300"
+              borderRadius="sm"
+              padding="sm"
+            >
+              Lorem ipsum dolor sit amet
+            </Box>
+          )
+        }
+      >
+        Show Toast
+      </Button>
+    )
+  }}
+</Playground>
+
+## Options
+
+You can choose between 2 options :
+
+- **position** : one of `top-left`, `top`, `top-right`, `bottom-left`, `bottom` (default) or `bottom-right`
+- **duration** : before closing element, in ms, with default `3000`. Can also be set to `null` if not automatic close your element.
+
+<Playground>
+  {() => {
+    const toast = useToast()
+    return (
+      <>
+        {['top-left', 'top', 'top-right', 'bottom-left', 'bottom', 'bottom-right'].map(position => (
+          <Button
+            key={position}
+            onClick={() =>
+              toast(
+                <Box
+                  backgroundColor="light.900"
+                  borderWidth="1px"
+                  borderStyle="solid"
+                  borderColor="nude.300"
+                  borderRadius="sm"
+                  padding="sm"
+                >
+                  Lorem ipsum dolor sit amet
+                </Box>,
+                { position, duration: 6000 }
+              )
+            }
+          >
+            {position}
+          </Button>
+        ))}
+      </>
+    )
+}}
+
+</Playground>
+
+## Snackbar
```

Comment:
## Supplied components

You can create your own notification component to display in the Toast or one of our default components: `Snackbar` and `Growl`. 

In general…
- a `Snackbar` is used to display a temporary, information-only notification, and 
- a `Growl` is used to display a more important notification with an action (that requires the user to close the notification)

### Snackbar

…